config/routes.rb

```diff
@@ -1,3 +1,6 @@
+require 'resque_web'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.